### PR TITLE
Added Italian language

### PR DESCRIPTION
**Pull request by ZinRicky**

### CODE REVIEW

**Comment by staniel359** [comment]
@ZinRicky Actually this one covers only Electron tranlslations (tray and update dialog).
For the whole application there're locales list [here](https://github.com/staniel359/muffon/blob/main/src/helpers/data/locales.js) and translation files [here](https://github.com/staniel359/muffon/tree/main/src/helpers/data/plugins/i18n/locales).
Are you going to continue your work?

**Comment by ZinRicky** [comment]
I’m sorry I didn’t see those 😅. By the way, I’m planning to commit the remaining translations this Sunday @ 22:00 UTC+2

**Comment by staniel359** [comment]
@ZinRicky Okay, looking forward to this.

**Comment by ZinRicky** [comment]
@staniel359 it should be done now.

File: electron/actions/i18n/setup.js [hunk 1]
```diff
@@ -20,12 +23,14 @@ const defaultLocale =
 const i18nData = {
   locales: [
     'en',
-    'ru'
+    'ru',
+	'it'
```
Comment (staniel359) [inline electron/actions/i18n/setup.js]: @ZinRicky Please fix indentation here.

File: electron/actions/i18n/setup.js [hunk 2]
```diff
@@ -20,12 +23,14 @@ const defaultLocale =
...
   ],
   defaultLocale,
   staticCatalog: {
     en,
-    ru
+    ru,
+	it
```
Comment (staniel359) [inline electron/actions/i18n/setup.js]: @ZinRicky Please fix indentation here.

File: src/helpers/data/locales.js [hunk 3]
```diff
@@ -6,5 +6,9 @@ export default [
   {
     code: 'ru',
     text: 'Русский'
+  },
+  {
+    code: 'it',
+	text: 'Italiano'
```
Comment (staniel359) [inline src/helpers/data/locales.js]: @ZinRicky Please fix indentation here.

Comment (staniel359) [reply src/helpers/data/locales.js]: @ZinRicky Also please place Italian after English, they should come alphabetically.

**Review by staniel359 — CHANGES_REQUESTED** [review]
@ZinRicky Please address these changes.

**Comment by staniel359** [comment]
@ZinRicky Also please squash your commits into a single one.

**Review by staniel359 — APPROVED** [review]
@ZinRicky Great job, thank you.

**Comment by staniel359** [comment]
@ZinRicky Could you squash the commits into one and name it `Add Italian language`?

**Comment by ZinRicky** [comment]
> Could you squash the commits into one and name it `Add Italian language`?

I’ve never done squashing, so I’m looking up online the best way to do it 😅

**Pull request closed without merging**